environment_test.yml

```diff
@@ -36,3 +36,10 @@ dependencies:
   - pyvista
   - pip
   - python-kaleido
+  # Linters and code style
+  - black
+  - flake8
+  - flake8-bugbear
+  - flake8-builtins
+  - flake8-mutable
+  - flake8-rst-docstrings
```

Comment:
we might also want to use `flake8-docstrings` for numpy style documentation checking
https://pypi.org/project/flake8-docstrings/